### PR TITLE
Use the same mechanism for the editor timeline as the embed timeline

### PR DESCRIPTION
**Pull request by vied12**

### CODE REVIEW

File: client/spec/editor_draft_posts_spec.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ describe('Draft Posts', function() {
             // timeline
             expect(element(by.css('.column-timeline')).element(by.repeater('post in posts').row(0)).isPresent()).toBe(true);
             // draft posts
-            expect(element(by.css('.column-draft-posts')).all(by.repeater('post in postsList.posts')).count())
+            expect(element(by.css('.column-draft-posts')).all(by.repeater('post in postsList.pagesManager.allPosts()')).count())
```
Comment (nistormihai) [inline client/spec/editor_draft_posts_spec.js]: this really proves the need for jasmin page object.

**Comment by nistormihai** [comment]
LGTM

File: client/app/scripts/liveblog-edit/pages-manager.service.js [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+                return retrievePage(1, to_page * self.maxResults).then(function(posts) {
+                    createPagesWithPosts(posts._items);
+                    return posts;
+
+                });
+            }
+
+            /**
+             * Returns the page index and the post index of the given post in the local pages
+             * @param {Post} post_to_find - post to find in the pages
```
Comment (nistormihai) [inline client/app/scripts/liveblog-edit/pages-manager.service.js]: What is this Post class?

Comment (vied12) [inline client/app/scripts/liveblog-edit/pages-manager.service.js]: It's a liveblog post object

**Pull request closed without merging**